operations/_scripts/deploy/deploy.sh

```diff
@@ -22,8 +22,20 @@ export TF_STATE_BUCKET="$(/bin/bash $GITHUB_ACTION_PATH/operations/_scripts/gene
 # Generate app repo
 /bin/bash $GITHUB_ACTION_PATH/operations/_scripts/generate/generate_app_repo.sh
 
-# Generate bitops config
-/bin/bash $GITHUB_ACTION_PATH/operations/_scripts/generate/generate_bitops_config.sh
+# Configure Ansible
+# Ansible related generation/resourcing/configurations block
+# -------------------------------- #
+if [[ -n $ST2_ANSIBLE_VARS_FILE ]]; then
+    cp $GITHUB_WORKSPACE/$ST2_ANSIBLE_VARS_FILE $GITHUB_ACTION_PATH/operations/deployment/ansible/vars/
+    export ST2_CONF_FILE=$(basename $GITHUB_WORKSPACE/$ST2_ANSIBLE_VARS_FILE)
+fi
+# -------------------------------- #
```

Comment:
maybe move to a dedicated script so it can be included in the common stuff

Reply:
It wasn't removed, it was just moved into its own tool named folder.
The script is still invoked beneath this section though